src/views/Dashboard.vue

```diff
@@ -385,7 +395,7 @@ export default {
 
   methods: {
     goToStandaloneDirectors () {
-      this.$router.push({ name: Routes.STANDALONE_DIRECTORS, params: { filingId: 0 } }) // 0 means "new COD filing"
+      this.$router.push({ name: Routes.STANDALONE_DIRECTORS, params: { filingId: '0' } }) // 0 means "new COD filing"
```

Comment:
Parameter is now expected to be a string.

Reply:
will this affect any other applications and do we need to change those as well?  I see you have already changed 2 here :)

Reply:
The old code worked fine and didn't show any errors/warnings in VS Code. I think the new code works, too, but VS Code complained about it, so I fixed it.

Since you asked, I checked Create UI and Edit UI, and they either don't pass URL parameters this way or they are already strings, so I think there's nothing more to update at this time.

The take-away here is that, as our tools improve (eg, Lint, language services), more code issues will be flagged, and we should try to fix those to prevent issues. Thanks for asking 👋